tests_disabled.json

```diff
@@ -203,7 +199,7 @@
         },
         {
             "name": "cache.test_cache.H2Cache",
-            "reason": "Disabled by issue #1669"
+            "reason": "Disabled by issue #1669 *1669 may not relates to current test*"
```

Comment:
Do we need a separate issue or will #1820 leave #1669 open?

Reply:
1669 will be closed. However, `cache.test_cache.H2Cache` still not works for another reason.

Reply:
Could you please create an issue for this in `tempesta-test` then?